python/cog/server/http.py

```diff
@@ -240,7 +242,21 @@ async def healthcheck() -> Any:
         else:
             health = app.state.health
         setup = attrs.asdict(app.state.setup_result) if app.state.setup_result else {}
-        return jsonable_encoder({"status": health.name, "setup": setup})
+        activity = runner.activity_info()
+        return jsonable_encoder(
+            {"status": health.name, "setup": setup, "concurrency": activity}
+        )
+
+    @app.get("/ready")
+    async def ready() -> Any:
```

Comment:
What does this do that the healthcheck endpoint doesn't?

Reply:
it's suitable for use as a readiness check with e.g. fly, since healthcheck returns 200 during setup or while predictions are running